File: server/src/main/java/br/com/projuris/profeed/dto/FeedBackDTO.java
package br.com.projuris.profeed.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FeedBackDTO {

    private Long id;

    private String text;

    private Long idFrom;

    private String nameFrom;

    private String photoUrlFrom;

    private Long idTo;

    private String nameTo;

    private String photoUrlTo;

}

File: mobile/src/app/domain/feedback-dto.ts
export class FeedbackDTO {

    id:number;
    nameFrom : string;
    nameTo : string;
    photoUrlFrom: string;
    photoUrlTo: string;
    text: string;

    constructor(id:number, nameFrom : string, nameTo : string,  photoUrlFrom: string, photoUrlTo: string, text:string){
        this.id = id;
        this.nameFrom = nameFrom;
        this.nameTo = nameTo;
        this.photoUrlFrom = photoUrlFrom;
        this.photoUrlTo = photoUrlTo;
        this.text = text;
    }
}
